docs/set.md

```diff
@@ -22,7 +25,38 @@ odf set recovery-profile high_client_ops
 
 To verify the recovery profile setting run [odf get recovery-profile](get.md#recovery-profile).
 
+## full
```

Comment:
One more request... It would be helpful in the examples to show ratios that are likely to be used, as well as document the defaults that we have in ODF. So if the defaults are .75/.80/.85 for nearfull/backfillfull/full, then how about we use example ratios of 0.05 higher at .80/.85/.90?

Reply:
done